python-sdk/tests_integration/databases/test_all_databases.py

```diff
@@ -118,15 +118,15 @@ def test_export_table_to_pandas_dataframe_mssql(
     [
         {"database": Database.BIGQUERY},
         {"database": Database.POSTGRES},
-        {"database": Database.REDSHIFT},
+        # {"database": Database.REDSHIFT},
```

Comment:
See https://github.com/astronomer/astro-sdk/pull/1807/files#r1118616324

Reply:
Added `-k not redshift` for integration test cases.